shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/rulealtered/RuleAlteredJob.java

```diff
@@ -80,7 +80,9 @@ public void execute(final ShardingContext shardingContext) {
             return;
         }
         log.info("start RuleAlteredJobScheduler, jobId={}, shardingItem={}", jobId, shardingItem);
-        RuleAlteredJobScheduler jobScheduler = new RuleAlteredJobScheduler(jobContext);
+        RuleAlteredJobScheduler jobScheduler = new RuleAlteredJobScheduler(jobContext, jobContext.getJobProcessContext());
```

Comment:
It might be not necessary to add `jobContext.getJobProcessContext()`, since we could get PipelineProcessContext from PipelineJobContext

Reply:
We could not move `PipelineProcessContext` to core module, since it's referenced in `PipelineJobContext`.

Since `ExecuteEngine` is in core module, could we ignore `PipelineProcessContext` and add inventory incremental execute engine?